business_rules/actions.py

```diff
@@ -45,8 +51,17 @@ def _validate_action_parameters(func, params):
 def rule_action(label=None, params=None):
     """
     Decorator to make a function into a rule action.
-
-    NOTE: add **kwargs argument to receive Rule and Matched Conditions as parameters in Action function
+    `params` parameter could be one of the following:
+    1. Dictionary with params names as keys and types as values
+    Example:
+    params={
+        'param_name': fields.FIELD_NUMERIC,
+    }
+
+    2. If a param has a default value, ActionParam can be used. Example:
+    params={
+        'voucher_template_id': ActionParam(field_type=fields.FIELD_NUMERIC, default_value=123)
```

Comment:
Could you please replace all use of voucher template id in docstring examples with generic ones?